### DISCUSSION TITLE
Redmi note 10 pro

### DISCUSSION THREAD

**Discussion by Bossplay228** [skyline-emu/skyline#1507] [Q&A]
why just shapes and beats does not work on Redmi Note 10 Pro, but this game is already working in this emulator, when I start the game sometimes it shows a black screen and the fps stops and that's it, but sometimes the beginning is shown and then the same problem occurs
![Screenshot_2022-06-05-13-28-12-069_skyline emu](https://user-images.githubusercontent.com/93477799/172896236-974846f4-90f0-4325-990d-26915835ff3c.jpg)
![Screenshot_2022-06-08-23-55-13-665_skyline emu](https://user-images.githubusercontent.com/93477799/172896267-2c7cddef-5804-4cbe-9651-83a824da4811.jpg)
![Screenshot_2022-06-09-18-21-46-778_skyline emu](https://user-images.githubusercontent.com/93477799/172896412-ab091ba0-e905-4f6a-b14f-68e98ea49714.jpg)

**Comment by yhak22** [comment]
Well the game isn't playable yet.
Wait for more updates. ;)

**Reply by Bossplay228** [reply]
strange, here is the link where there is proof: https://github.com/skyline-emu/skyline-games-list/issues/535

**Reply by Bossplay228** [reply]
maybe someone has a special phone that he was able to run this game
